### DISCUSSION TITLE
Repository Not be transfer to another username

### DISCUSSION THREAD

**Discussion by Rohan-Kar** [community/community#38086] [New to GitHub]
I have tried to transfer the repository to another user name but after the 2 days it will not transfer and it will show the the abort the transfer in the setting section . Please guide e on this .

**Comment by ossanna16** [comment]
Hi there @Rohan-Kar and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
